src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1016,11 +1018,13 @@ InferenceProfiler::Measure(
   previous_window_end_ns_ = window_end_ns;
 
   RETURN_IF_ERROR(manager_->GetAccumulatedClientStat(&end_stat));
+  prev_client_side_stats_ = end_stat;
 
   // Get server status and then print report on difference between
   // before and after status.
   if (include_server_stats_) {
     RETURN_IF_ERROR(GetServerSideStatus(&end_status));
```

Comment:
isn't this supposed to be moved closer to where the end timestamp is?

Reply:
also, above, the server is fetched first, then client, but down here, the client is fetched first, then server. any rationale behind this ordering?